dockers/docker-database/create_all_redis_config

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/python
+import json
+import os
+
+database_config_file = "/var/run/redis/database_config.json"
```

Comment:
>/var/run/redis/ [](start = 24, length = 15)

This directory is used by vanilla redis-server, suggest to create a new one.

Reply:
modified  as required